kettle-columnstore-bulk-exporter-plugin/src/main/java/com/mariadb/columnstore/api/kettle/KettleColumnStoreBulkExporterStep.java

```diff
@@ -329,7 +329,12 @@ public boolean processRow( StepMetaInterface smi, StepDataInterface sdi ) throws
                         }
                     }else{
                         logDebug("Try to insert item " + i + " as Timestamp");
-                        java.sql.Timestamp t = valueMetaTimestamp.getTimestamp(r[i]);
+                        java.sql.Timestamp t;
+                        if (data.rowValueTypes.get(i).getNativeDataType(r[i]) instanceof java.sql.Timestamp){
+                            t = (java.sql.Timestamp) data.rowValueTypes.get(i).getNativeDataType(r[i]);
```

Comment:
Might be a good idea to call this once, not sure if the call is expensive but might also make the code more readable.